mersenne-31/src/complex.rs

```diff
@@ -33,6 +35,18 @@ impl<AF: AbstractionOf<Mersenne31>> Mersenne31Complex<AF> {
     pub fn imag(&self) -> AF {
         self.parts[1].clone()
     }
+
+    pub fn conjugate(&self) -> Self {
+        Self::new(self.real(), self.imag().neg())
+    }
+
+    pub fn magnitude(&self) -> AF {
```

Comment:
I think technically it should be called magnitude squared?

Reply:
fixed